app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -50,6 +51,14 @@ public function panel(Panel $panel): Panel
                 'warning' => Color::Amber,
                 'blurple' => Color::hex('#5865F2'),
             ])
+            ->renderHook('panels::sidebar.footer', fn () => view('filament.footer')) // TODO make the footer also work with topNavigation
```

Comment:
```suggestion
            ->renderHook(PanelsRenderHook::FOOTER, fn () => view('filament.footer')) // TODO make the footer also work with topNavigation
```

Reply:
When you do this with only footer and not sidebar footer then you have this:

![image](https://github.com/pelican-dev/panel/assets/62171904/aad8be8d-1f08-4a48-b0d0-4c28881d8f6c)

I have tested almost all of the renderhooks and this is the only one that comes out nice:

![image](https://github.com/pelican-dev/panel/assets/62171904/b5741303-c678-48b6-a294-723809b5c37d)